app/javascript/components/widgets/numbers/ducks.jsx

```diff
@@ -27,51 +23,29 @@ export const withLocalMutation = WrappedComponent => props => (
 );
 
 export const nullState = {
-  githubPull: 0,
-  githubCommit: 0,
-  slackMessage: 0,
   world: null,
 };
 
 export const getState = ({
-  fallingBlocksState: {
-    githubCommit = 0,
-    githubPull = 0,
-    slackMessage = 0,
-    world = null,
-  } = {},
+  fallingBlocksState: { world = null } = {},
 } = {}) => {
-  if (!world) {
+  if (!world || typeof world !== 'object') {
```

Comment:
Nit -- took me a couple reads to understand this -- the `typeof world !== 'object'` is there for some type checking? If so, maybe `Array.isArray(world)` might be more clear (and would obviate the need for `!world ||`).

Reply:
Yeah, that's better. This was in here to safe guard against the old zlib data from sneaking in.